chemprop/cli/predict.py

```diff
@@ -307,12 +340,7 @@ def main(args):
 
     model_paths = find_models(args.model_path)
```

Comment:
Now with the alias, we can change to model_paths
```suggestion
    model_paths = find_models(args.model_paths)
```